### PR TITLE
Add Clear Mapping Co to commercial_support.rst

### PR DESCRIPTION
**Pull request by nickbearman**
Hi,

I'd like to add Clear Mapping Co to the commercial support page in this PR.

We're not a Core Contributors, but what are the criteria for Contributors?

We have sponsored a QGIS South West User Group meeting (http://www.clearmapping.co.uk/our-blog/item/457-qgis-south-west-user-group-event.html), and I (@nickbearmanuk) have made a small contribution to the QGIS Documentation a while a go. 

If we qualify for Contributors, great - I can update the PR. Otherwise we are very happy to be listed under 'Other'.

Best wishes,
Nick.

### CODE REVIEW

**Comment by nickbearman** [comment]
Hi, the Travis CI check failed, but it flagged a file I haven't touched - /home/travis/build/qgis/QGIS-Website/source/site/about/case_studies/poland_ffth.rst
Is there anything I can do about this?
Sorry!
Best wishes,
Nick.

**Comment by DelazJ** [comment]
Yes, website build was broken. Fixed now so reopening your PR should trigger a check only on your changes.

File: source/site/forusers/commercial_support.rst [hunk 1]
```diff
@@ -230,6 +230,15 @@ provides installation, training, and implementation support for QGIS.
 
 ----
 
+.. |clearmapping| image:: /static/site/about/images/clear_mapping_co.jpg
```
Comment (yjacolin) [inline source/site/forusers/commercial_support.rst]: it should be `|clear_mapping_co|` here or change below ;)

**Review by yjacolin — CHANGES_REQUESTED** [review]

**Comment by nickbearman** [comment]
Thanks @yjacolin for spotting the mistake - I have corrected it.

**Comment by nickbearman** [comment]
Thanks @DelazJ for correcting the build process - all looks good now.

**Comment by DelazJ** [comment]
Credits of build fixes go to @yjacolin and @rduivenvoorde . Thanks to them. I was simply the messenger :)

**Comment by nickbearman** [comment]
Hi @yjacolin I think I've done all the changes you requested, do let me know if I have missed something!

**Pull request merged**